onnx/test/cpp/parser_test.cc

```diff
@@ -488,6 +488,34 @@ square (x) => (y) {
 )ONNX";
 
   CheckModel(code);
+
+  const char* code_function_with_attributes = R"ONNX(
+<
+  ir_version: 9,
+  opset_import: [ "" : 15, "custom_domain" : 1]
+>
+agraph (float[N] x) => (float[N] out)
+{
+  out = custom_domain.foo<alpha=2.0, gamma=3.0>(x)
```

Comment:
We should also have an example with any value specified for alpha.

Reply:
You mean an example with no value specified for alpha? Yes, agree. (But the important step is to extend the checker and inference methods to support this. It will mostly work fine for parsing, but not for checking/inference.)

Reply:
added test for combination of with or without values.